source/_integrations/zwave_js.markdown

```diff
@@ -55,6 +55,51 @@ Advanced users: Make sure that the server started successfully by inspecting the
 
 ## Services
 
+### Service `zwave_js.set_config_parameter`
+
+This service will update a configuration parameter. At this time, it is not possible to update multiple partial parameters in a single call, but we hope to add support for that in the future.
+
+| Service Data Attribute 	| Required  	| Description                                                                                                                               	|
+|------------------------	|-----------	|-------------------------------------------------------------------------------------------------------------------------------------------	|
+| `entity_id`            	| Exclusive 	| Entity (or list of entities) to set the configuration parameter on. Either this or `device_id` must be provided.                       	|
```

Comment:
We changed it and entity_id and device_id are no longer exclusive.

Reply:
thanks for catching that